site/pages/core/authenticate-users.mdx

```diff
@@ -50,6 +50,38 @@ await signer.authenticate({ type: "email", bundle });
 
 :::
 
+## Social Login
+
+You can allow uers to sign in with a Oauth provider like Google, Facebook, or Auth0. There are two login flows that are possible: the redirect flow goes to the auth provider's login page and the popup flow has the auth provider login screen to open as a popup.
+
+:::code-group
```

Comment:
this code block isn't rendering in the vercel preview for some reason https://aa-sdk-site-git-alex-oauth-docs-alchemy-dot-com.vercel.app/core/authenticate-users